src/main/java/wolox/training/models/Book.java

```diff
@@ -121,6 +135,7 @@ public String getIsbn() {
     }
 
     public void setIsbn(String isbn) {
+        Preconditions.checkNotNull(isbn, "Isbn cannot be null");
```

Comment:
In addition to the change suggested above, we could add a check that verifies that the content of this string is numeric

`.checkArgument (StringUtils.isNumeric (isbn), "error message");`

Reply:
Add new validate to set